### ISSUE TITLE
NotImplementedError: Could not run 'aten::gelu.out' with arguments from the 'ocl' backend.

### ISSUE THREAD

**Issue by zougloub** [artyom-beilis/pytorch_dlprim#19]
Implementing `F.gelu()`.

**Comment by artyom-beilis** [comment]
Yes this is one of gaps I'm going to close in near future implementing transformers

https://github.com/artyom-beilis/pytorch_dlprim/discussions/16

**Comment by zougloub** [comment]
ELU, [GELU](https://arxiv.org/pdf/1606.08415v3.pdf), SiLU, [Swish](https://arxiv.org/pdf/1710.05941.pdf) can be used in pretty much in any context.

**Comment by artyom-beilis** [comment]
Fixed in f2fff47b49dc9068ca90be72bea23d2547fb1c4b
